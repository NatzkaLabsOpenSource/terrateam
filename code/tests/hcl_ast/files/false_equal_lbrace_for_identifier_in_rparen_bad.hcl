false = { for foo in )
---
false = { for foo in ]
---
false = { for foo in *
---
false = { for foo in +
