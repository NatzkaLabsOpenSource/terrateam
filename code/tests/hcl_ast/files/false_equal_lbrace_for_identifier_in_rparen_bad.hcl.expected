Expected expression in "in" section of object for definition.
